doc/source/whatsnew/v1.5.0.rst

```diff
@@ -786,6 +786,8 @@ Indexing
 - Bug in :meth:`Series.asof` and :meth:`DataFrame.asof` incorrectly casting bool-dtype results to ``float64`` dtype (:issue:`16063`)
 - Bug in :meth:`NDFrame.xs`, :meth:`DataFrame.iterrows`, :meth:`DataFrame.loc` and :meth:`DataFrame.iloc` not always propagating metadata (:issue:`28283`)
 - Bug in :meth:`DataFrame.sum` min_count changes dtype if input contains NaNs (:issue:`46947`)
+- Bug in :class:`IntervalTree` that lead to an infinite recursion. (:issue:`46658`)
```

Comment:
Since `IntervalTree` is not a public object, could you instead mention the scenario in your original issue in which you encountered the bug (Index with interval type and `get_indexer_for`)?

Reply:
Doesn't appear that your latest push changed this entry, but happy to have it when you get the chance next week.